error.go

```diff
@@ -4,7 +4,9 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
-var InvalidConfigError = microerror.New("invalid config")
+var InvalidConfigError = &microerror.Error{
+	Kind: "InvalidConfigError",
+}
```

Comment:
Two things. 

- The files are called `error.go`. My script didn't work and I was wondering why. Yeah guess why. This is why consistency matters. 
- IMO the error should be private. AFAIK I also mentioned this in my PR review. I do not see why it has to be public and this is actually a sign of design smell IMO.